src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -519,6 +518,28 @@ export class BrowsingContextImpl {
     };
   }
 
+  async reload(
+    ignoreCache: boolean,
+    wait: BrowsingContext.ReadinessState
+  ): Promise<Message.EmptyResult> {
+    await this.#cdpTarget.cdpClient.sendCommand('Page.reload', {
+      ignoreCache,
+    });
+
+    switch (wait) {
+      case 'none':
+        break;
+      // TODO: handle interactive.
+      case 'interactive':
+        break;
+      // TODO: handle complete.
+      case 'complete':
+        break;
```

Comment:
Can we return "not implemeted" error instead?

Reply:
Done